src/AppBundle/Subscriber/AutoLabelPRFromContentSubscriber.php

```diff
@@ -109,7 +109,7 @@ private function getValidLabels()
             'Asset', 'BC Break', 'BrowserKit', 'Bug', 'Cache', 'ClassLoader',
             'Config', 'Console', 'Critical', 'CssSelector', 'Debug', 'DebugBundle',
             'DependencyInjection', 'Deprecation', 'Doctrine', 'DoctrineBridge',
-            'DomCrawler', 'Drupal related', 'DX', 'Easy Pick', 'Enhancement',
+            'DomCrawler', 'DotEnv', 'Drupal related', 'DX', 'Easy Pick', 'Enhancement',
```

Comment:
This PR should be merged with the upcoming 3.3 release, and we also need "Image", "Lock" and "UTF-8" :).